### DISCUSSION TITLE
part-ii/2018-73/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#88] [Question comments]
Part II 2018 General Relativity: Paper 4, Section II, E 

https://questions.tripos.org/part-ii/2018-73/

**Comment by AndoBando** [comment]
If you treat all the quantities they tell you are small as of order $\varepsilon$, you can get some terms of order $\varepsilon^{3/2}$ involving time derivatives of $\Phi$. It's fine to ignore these. 
In the end $S/T \to 1$. "The local speed of light"
